.devcontainer.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Could we move this to `.devcontainer/devcontainer.json`? That's the pattern we've used in other repos (e.g. https://github.com/Azure/bicep/blob/main/.devcontainer/devcontainer.json)